package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-api-types",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Ha, no.
```suggestion
  "version": "0.5.1",
```

Reply:
npm automatically changed this because it's 0.6.0 in package.json

Reply:
Oh pfft, it's the lock file
I'm blind